<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NP-SADEKUNLE" name="Bingo_Core_SUITE.RegistrationTest" tests="10" failures="0" timestamp="8 Mar 2017 09:42:19 GMT" time="411.855" errors="1">
  <testcase name="RegistrationTest" time="39.728" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="38.898" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="40.037" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="42.011" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="38.820" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="58.110" classname="Bingo_Core_SUITE.RegistrationTest">
    <error type="java.lang.AssertionError" message="expected [FAILURE] but found [SUCCESS]">
      <![CDATA[java.lang.AssertionError: expected [FAILURE] but found [SUCCESS]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at Bingo_Core_SUITE.RegistrationTest.RegistrationTest(RegistrationTest.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- RegistrationTest -->
  <testcase name="RegistrationTest" time="36.165" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="37.568" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="41.657" classname="Bingo_Core_SUITE.RegistrationTest"/>
  <testcase name="RegistrationTest" time="38.861" classname="Bingo_Core_SUITE.RegistrationTest"/>
</testsuite> <!-- Bingo_Core_SUITE.RegistrationTest -->
